### ISSUE TITLE
HEAD doesn't build

### ISSUE THREAD

**Issue by andresilva** [pigoz/homebrew-mplayer2#7]
Hi,

I've tried to build the latest version of the formula with ffmpeg and got the following error:

```
==> Cloning git://git.mplayer2.org/mplayer2.git
Updating /Library/Caches/Homebrew/mplayer2--git
==> Generating VERSION from the Homebrew's git cache
==> ./configure --prefix=/usr/local/Cellar/mplayer2/HEAD --cc=cc --enable-macosx-bundle --enable-macosx-finder --enable-apple-remote
==> make install
    int (*encode2)(AVCodecContext *avctx, AVPacket *avpkt, const AVFrame *frame,
          ^
1 error generated.
make: *** [libmpcodecs/vf_lavc.o] Error 1
make: *** Waiting for unfinished jobs....

Error: mplayer2 did not build
Logs: /Users/beat/Library/Logs/Homebrew/mplayer2/
Help: https://github.com/mxcl/homebrew/wiki/troubleshooting
```

I'm using OSX Mountain Lion (10.8.2) and have the latest Xcode version (4.5.1).

**Comment by pigoz** [comment]
Hello Andre, thank you for reporting. Your problem should be fixed with the commit 1ae939dcc I just pushed.

You may have to clear homebrew's git cache of mplayer2 located at `brew --cache`, it sometimes gives me problems when I switch branches.

**Comment by andresilva** [comment]
Yep, it's working now. Thanks!

I had to install docutils since the install requires rst2man.
